build.gradle

```diff
@@ -7,6 +7,7 @@ plugins {
 
 allprojects {
     apply plugin: 'idea'
+    apply plugin: 'eclipse'
```

Comment:
Doug added this and I was wondering if we should remove or leave for eclipse users.

Reply:
I would say delete it, but I don't think it particularly matters.